src/physfs/util.cpp

```diff
@@ -60,6 +60,21 @@ bool remove(const std::string& filename)
   return PHYSFS_delete(filename.c_str()) == 0;
 }
 
+void
+remove_with_content(std::string& filename)
```

Comment:
I think you can mark the filename reference `const`

Reply:
Or does that lead to conflicts with line 75?

Reply:
In any case, the caller does not need to have the string edited, so it should be passed by `const&` and copied inside the function if necessary.